src/views/List.jsx

```diff
@@ -53,42 +53,65 @@ export function List({ data, listToken, isLoading }) {
 		}
 	};
 	return (
-		<>
+		<div className="min-h-[100vh]">
 			{isLoading ? (
 				'Loading...'
 			) : data.length ? (
 				<>
 					{listToken && (
-						<div>
+						<div className="flex items-center justify-center gap-2 lg:gap-4 text-[15px] lg:text-[22px] my-3">
 							<p>Your list : {listToken}</p>
```

Comment:
I don't think setting text sizes here is necessary.

I think the text on the list is too large. This is what I see
![image](https://user-images.githubusercontent.com/74952593/204005746-a887b712-693c-4b66-b280-66c80120e5a4.png)

If I remove it from "your list" text, this is how it changes
![image](https://user-images.githubusercontent.com/74952593/204005998-680552be-b12d-49dc-9f73-b72f3b25bfa1.png)

Reply:
Though no much difference, so it's ok to remove the specified text sizes from the list.